manifests/m/MariaDB/Server/11.3.2/MariaDB.Server.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: MariaDB.Server
+PackageVersion: 11.3.2

```

```suggestion
PackageVersion: 11.3.2.0
```
Must match version written to Control Panel / Apps&Features